wfms/camunda/v1/WfMSPlugin.java

```diff
@@ -0,0 +1,77 @@
+package cloud.benchflow.plugins.camunda.v1;
```

Comment:
Since you are there, use the actual version range supported for sure but this plugin: 7.3.0 - 7.5.0